### ISSUE TITLE
Docker --version broken on 19.03.0 (bottled)

### ISSUE THREAD

**Issue by ms-ati** [Homebrew/homebrew-core#42351]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [x] are reporting a bug others will be able to reproduce and not asking a question or requesting software. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh. To get software added or changed in Homebrew please file a [Pull Request](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)
- [x] have a problem with `brew install` (or `upgrade`, `reinstall`) a single, Homebrew/homebrew-core formula (not cask) on macOS? If it's a general `brew` problem please file this issue at Homebrew/brew: https://github.com/Homebrew/brew/issues/new/choose. If it's a Linux problem please file this issue at https://github.com/Homebrew/linuxbrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [x] ran `brew update` and can still reproduce the problem?
- [x] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [x] ran `brew gist-logs <formula>` (where `<formula>` is the name of the formula that failed) and included the output link?
```
$ brew gist-logs docker
Error: No logs.
```

- [x] if `brew gist-logs` didn't work: ran `brew config` and `brew doctor` and included their output with your issue?
```
$ brew config
HOMEBREW_VERSION: 2.1.8
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 549a40bedb32171912a449d8d0e7bcae02d5b432
Last commit: 2 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: afd1f699d17314a985ac50121b26df519cf1e4a5
Core tap last commit: 20 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_DEV_CMD_RUN: 1
HOMEBREW_GITHUB_API_TOKEN: set
CPU: octa-core 64-bit ivybridge
Homebrew Ruby: 2.3.7 => /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/bin/ruby
Clang: 10.0 build 1001
Git: 2.22.0 => /usr/local/bin/git
Curl: 7.54.0 => /usr/bin/curl
Java: 1.8.0_102, 1.8.0_66, 1.8.0_60, 1.8.0_51, 1.8.0_31, 1.7.0_25
macOS: 10.14.6-x86_64
CLT: 10.3.0.0.1.1562985497
Xcode: 10.3
CLT headers: 10.3.0.0.1.1562985497
```

```
$ brew doctor
Your system is ready to brew.
```

To help us debug your issue please explain:

- What you were trying to do (and why)
**What:** Retrieving the **Docker version** from the command line as follows:

```
$ docker --version
Docker version unknown-version, build unknown-commit
```

*Why:* Investigating why the [dinghy](https://github.com/codekitchen/homebrew-dinghy/blob/master/Formula/dinghy.rb) formula was broken after running `brew upgrade` today (Tue July 23 2019). Error is as follows:

```
$ dinghy
/usr/local/Cellar/dinghy/4.6.5/cli/dinghy.rb:100:in `version': undefined method `[]' for nil:NilClass (NoMethodError)
	from /usr/local/Cellar/dinghy/4.6.5/cli/dinghy.rb:78:in `block in version_check'
	from /usr/local/Cellar/dinghy/4.6.5/cli/dinghy.rb:77:in `each'
	from /usr/local/Cellar/dinghy/4.6.5/cli/dinghy.rb:77:in `version_check'
	from /usr/local/Cellar/dinghy/4.6.5/cli/dinghy.rb:43:in `run_checks'
	from /usr/local/bin/_dinghy_command:11:in `<main>'
```

- What happened (include command output)
Docker version is apparently misconfigured in the bottled executable, which I presume is breaking the `dinghy` tooling that parses it.

```
$ docker --version
Docker version unknown-version, build unknown-commit
```

- What you expected to happen
I expected it to return the expected parsable version string, perhaps `19.03.0`?

```
brew info docker

$ brew info docker
docker: stable 19.03.0 (bottled)
Pack, ship and run any application as a lightweight container
https://www.docker.com/
/usr/local/Cellar/docker/19.03.0 (8 files, 79.9MB) *
  Poured from bottle on 2019-07-23 at 14:35:13
From: https://github.com/Homebrew/homebrew-core/blob/master/Formula/docker.rb
==> Dependencies
Build: go ✘
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Analytics
install: 37,151 (30 days), 96,146 (90 days), 381,649 (365 days)
install_on_request: 30,748 (30 days), 82,346 (90 days), 325,520 (365 days)
build_error: 0 (30 days)
```

- Step-by-step reproduction instructions (by running `brew install` commands)

```
brew install docker
docker --version
```

**Comment by mediaessenz** [comment]
I can confirm this issue

**Comment by pkoltermann** [comment]
+1

**Comment by iMichka** [comment]
I think this should be fixed after #42326 is merged

**Comment by SMillerDev** [comment]
It's merged. Closing this issue
